nav2_msgs/action/ComputePathToPose.action

```diff
@@ -1,3 +1,15 @@
+# Error codes
+int16 NONE=0
+int16 UNKNOWN=1
+int16 START_OCCUPIED=2
+int16 GOAL_OCCUPIED=3
+int16 NO_VALID_PATH=4
+int16 TIMEOUT=5
+int16 START_OUTSIDE_MAP=6
```

Comment:
Seems like 6,7,8 could be put into use in the planners? I don't see them applied.

Reply:
No valid path and timeout gets throw by smac and hybrid a star. Start outside map gets throw by navfn.